src/switcher-ui.tsx

```
@@ -1804,14 +2256,228 @@ function SwitcherApp() {
                 outline: 'none',
               }}
             />
+            {/* Scope chips: live processes (issue #94) and saved lists
```

ESLint

[error] Insert `··`

(prettier/prettier)